popmon/visualization/histogram_section.py

```diff
@@ -189,3 +205,96 @@ def _plot_histograms(feature, date, hc_list, hist_names):
         plot = ""
 
     return {"name": date, "description": get_stat_description(date), "plot": plot}
+
+
+def _plot_heatmap(feature, date, hc_list) :
+    
+    hist_names = [' Heatmap ', ' Column Normalized Heatmap ',  ' Row Normalized Heatmap ' ]
```

Comment:
It would be nice if the user can disable certain versions of the heatmap if not needed to improve performance. Suggested to pass a config parameter from HistogramSection.